src/components/SwapWidget/SwapWidget.tsx

```diff
@@ -445,13 +445,9 @@ const SwapWidget = () => {
         // Setting quote amount prevents the UI from updating if pricing changes
         dispatch(setTradeTermsQuoteAmount(bestTradeOption!.quoteAmount));
         // Last look order.
-        //@ts-ignore
-        // TODO: give order the correct type
         const { order, senderWallet } = await LastLook.getSignedOrder({
           locator: bestTradeOption!.pricing!.locator,
           terms: { ...tradeTerms, quoteAmount: bestTradeOption!.quoteAmount },
-        }).catch((e) => {
```

Comment:
catching here is wrong - if we can't generate the order we need to hit the catch on line 486. It was also the source of the type error, because `order` could be undefined if `getSignedOrder` throws (the catch returns nothing)

Reply:
Yeah... I thought I had removed it previously when you commented on it the last time, but it sort of snuck back.